coralproject/talk
# Live Chat Epic

## What does this PR do?

Contains the entirety of deploy-able changes for the Live Chat.

All live chat features will be merged into this main epic branch. Once Live Chat iteration is complete, this epic branch will be merged into develop.

## What changes to the GraphQL/Database Schema does this PR introduce?

Flags and types required for Live Chat.

## How do I test this PR?

Will post full testing steps later after Live Chat has been completed, but this will get you started:

- Enable live chat on your tenant
- Create a new story with the live chat option enabled in the embed
- Navigate to the story and see live chat enabled
- Play around and try to break things!
